src/VisualStudio/VisualBasic/Impl/CodeModel/VisualBasicCodeModelService.vb

```diff
@@ -180,13 +180,17 @@ Namespace Microsoft.VisualStudio.LanguageServices.VisualBasic.CodeModel
                      SyntaxKind.SetAccessorBlock,
                      SyntaxKind.AddHandlerAccessorBlock,
                      SyntaxKind.RemoveHandlerAccessorBlock,
-                     SyntaxKind.RaiseEventAccessorBlock,
-                     SyntaxKind.DeclareFunctionStatement,
-                     SyntaxKind.DeclareSubStatement
+                     SyntaxKind.RaiseEventAccessorBlock
                     If scope = EnvDTE.vsCMElement.vsCMElementFunction Then
                         Return True
                     End If
 
+                Case SyntaxKind.DeclareFunctionStatement,
+                     SyntaxKind.DeclareSubStatement
+                    If scope = EnvDTE.vsCMElement.vsCMElementDeclareDecl Then
+                        Return True
```

Comment:
@pilchie An interesting point. Since Declare Function and Declare Sub are CodeElements of type `CodeFunctionDeclareDecl` which also happens to be `CodeFunction`, both Declare function and sub should also belong to the scope `EnvDTE.vsCMElement.vsCMElementFunction` 

Hence the condition above should be 
```VB
If scope = EnvDTE.vsCMElement.vsCMElementDeclareDecl OrElse
   scope = EnvDTE.vsCMElement.vsCMElementFunction
```

What do you think?